bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_utils.py

```diff
@@ -74,6 +81,82 @@ def _string_to_root_attribute(value, lookup_map):
         return value
 
 
+def _dict_to_std_map(arg_dict, allowed_val_dict):
+    """Helper function to convert python dict to std::map."""
```

Comment:
Please explain in the docstring the meaning of the parameters, in particular how `allowed_val_dict` is interpreted. It's probably best to explain this with a little example in the doc string.